lib/l10n/app_en.arb

```
@@ -312,6 +312,139 @@
+    "placeholders": {
+      "duration": {"type": "String"}
+    }
+  },
+  "processingError": "Processing Error",
+  "failedToProcessVideo": "Failed to process selected video: {error}",
+  "@failedToProcessVideo": {
+    "placeholders": {
+      "error": {"type": "String"}
+    }
+  },
+  "error": "Error",
```

RuboCop (1.76.1)

[warning] Duplicated key in hash literal.

(Lint/DuplicateHashKey)